core/src/main/java/com/xatkit/core/recognition/IntentRecognitionProviderFactory.java

```diff
@@ -35,6 +36,20 @@
  */
 public class IntentRecognitionProviderFactory {
 
+    /**
+     * The database model that will be used for this instance of Xatkit.
+     * <p>
+     * This property is optional, and it's default value is set to "MAPDB" if not specified
+     */
+    private static final String XATKIT_DATABASE_KEY = "xatkit.database.model";
+
+    /**
+     * The default value for xatkit.database.model in case it's not specified in the properties file.
+     */
+    private static final String DEFAULT_DATABASE = "MAPDB";
```

Comment:
Renaming it DEFAULT_DATABASE_MODEL.
I am also adding a new DATABASE_MODEL_MAPDB="mapdb" constant, this way we can easily initialize DEFAULT_DATABASE_MODEL = DATABASE_MODEL_MAPDB, and make suire it's easy to change the default value later.